Controller/ApiController.php

```diff
@@ -23,12 +23,11 @@
 use JMS\TranslationBundle\Translation\Updater;
 use Symfony\Component\HttpFoundation\Response;
 use JMS\TranslationBundle\Util\FileUtils;
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
 
 /**
- * @Route("/api", service="jms_translation.controller.api_controller")
+ * @Route("/api")
```

Comment:
This [is a deprecation for Route and Method from SensioFrameworkExtraBundle](https://symfony.com/doc/current/bundles/SensioFrameworkExtraBundle/annotations/routing.html#how-to-update-your-applications).